cmd/operator-sdk/printdeps/cmd.go

```diff
@@ -50,13 +49,20 @@ func printDepsFunc(cmd *cobra.Command, args []string) error {
 	}
 	projutil.MustInProjectRoot()
 
+	// Allow the cases of some sub-commands might not require check the deps files.
+	if projutil.GetOperatorType() == projutil.OperatorTypeUnknown {
+		err := fmt.Errorf("unable to print the deps because was not possible to identify the type of the project")
+		log.Fatal(err)
```

Comment:
rephrase the message "unknown project type, we were unable to print the deps"
also  if line length is not an issue I'd just do:

```
log.Fatal(fmt.Errorf("unknown project type, we were unable to print the deps"))
```

Reply:
👍